nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/audit/ProcessGroupAuditor.java

```diff
@@ -246,6 +252,48 @@ public void enableComponentsAdvice(ProceedingJoinPoint proceedingJoinPoint, Stri
         }
 
         saveUpdateProcessGroupAction(groupId, operation);
+        if (components.get(Component.Processor) != null) {
```

Comment:
It looks like `components.containsKey()` would be the more appropriate check for these three conditionals. It may also be simpler to create a static List of these three types and implement a simple loop.